tests/manual/texttransformation.md

```diff
@@ -17,17 +17,22 @@ Some of the transformations are:
     * Operators: `<=` to `≤`, `>=` to `≥`, `!=` to `≠`.
 
 1. Typography:
-    
+
     * Dashes: ` -- `, ` --- `.
     * Ellipsis: `...` to `…`
-    
+
 1. Quotes:
 
-    * Primary quotes (english): `'Foo bar'` to `‘Foo bar’` 
+    * Primary quotes (english): `'Foo bar'` to `‘Foo bar’`
     * Secondary quotes (english): `"Foo bar's"` to `“Foo bar's”`
 
 ### Testing
 
 * Check if the transformation works. Note that some might need a space to trigger (dashes).
 * Undo a text transformation and type - it should not re-transform it.
-* Change selection - the not transformed elements should stay. 
+* Change selection - the not transformed elements should stay.
+
+### Behaviour inside the code blocks
```

Comment:
Where do those changes come from? The ticket is about not creating multuple instances of text watcher. I thought we resolved the code block issue previously.

Reply:
Right, I should have added this before... I will update the test in another PR (or maybe this can be added on `master` directly?).